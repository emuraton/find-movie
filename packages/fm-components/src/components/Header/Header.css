.Header {
  position: relative;
  top: 0;
  left: 0;
  width: 100%;
  background-color: #081c24;
}

.Header-content {
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.Header-content-span {
  flex: 1;
  color: #ffffff;
  margin-left: 20px;
  text-transform: uppercase;
}
