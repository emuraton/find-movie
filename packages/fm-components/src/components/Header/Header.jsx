import React from 'react'
import iconImg from './images/icon-header.png'

require('./Header.css')

export default () => {
  return (
    <header className="Header">
      <div className="Header-content">
        <img alt="icon-movie-db" src={iconImg} />
        <span className="Header-content-span">Find your movie</span>
      </div>
    </header>
  )
}
