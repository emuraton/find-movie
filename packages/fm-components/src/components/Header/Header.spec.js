import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Header from './Header'

describe('<Header />', () => {

  describe('@renders', () => {
    it('in default state', () => {
      const wrapper = shallow(<Header />)
      expect(toJson(wrapper))
        .toMatchSnapshot()
    })
  })
})
