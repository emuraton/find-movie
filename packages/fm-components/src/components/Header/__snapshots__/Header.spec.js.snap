// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> @renders in default state 1`] = `
<header
  className="Header"
>
  <div
    className="Header-content"
  >
    <img
      alt="icon-movie-db"
      src={
        Object {
          "process": [Function],
        }
      }
    />
    <span
      className="Header-content-span"
    >
      Find your movie
    </span>
  </div>
</header>
`;
